test/system/distr_tests.cc

```diff
@@ -122,7 +122,7 @@ namespace detail {
 	    "[reductions][print_graph][smoke-test]") //
 	{
 		env::scoped_test_environment test_env(print_graphs_env_setting);
-		// init runtime early so the distr_queue ctor doesn't override the log level set by log_capture
+		// init runtime early so the queue ctor doesn't override the log level set by log_capture
```

Comment:
This comment is outdated (we still need to initialize it early though, for the `runtime_testspy` calls below)

Reply:
Good catch, fixed!